http/1.6/modules/ROOT/pages/http-listener-ref.adoc

```diff
@@ -210,93 +333,135 @@ blue
-By default, a regular response returns a 200 status code and the message payload as the body, while an error response returns a 500 status code with the flow error's description as the body. These responses can be customized by providing the:
+== Sending HTTP Responses
 
-* Status code
-* Reason phrase
-* Headers multi-map
-* Body
+After the triggered flow finishes its execution, the HTTP *Listener* source sends an HTTP response with the result. If the flow executes successfully, the source sends an HTTP regular response. Otherwise, the source sends an HTTP error response.
+By default, an HTTP regular response returns a 200 status code and the message payload as the body, while an HTTP error response returns a 500 status code with the flow error's description as the body.
 
-DataWeave can be used to generate each parameter and variables can be used to propagate data from the flow.
+In the HTTP *Listener* source configuration, you can customize HTTP responses by providing the following parameters. Note that you can use DataWeave to generate content for each parameter, and variables to propagate data from the flow:
 
-In the following example, an endpoint is defined to store logos for an account:
+* *Body*
+* *Headers*
+* *Status code*
+* *Reason phrase*
 
-. When the storage is successful, a `201 Created` is returned with a body, such as `Corporate Logo has been stored as a MuleSoft logo`.
-. When the storage fails, the status code is defined through the `errorCode` variable (if available) or `500` by default.
-. A custom header is added (`X-Time`) as well.
-. A body, such as `Corporate Logo could not be stored`, is returned.
-. If there is a `CONNECTIVITY` error storing the logo, for example, the returned status code would be `504`, while any other errors would result in a `500`.
+=== Create Sending HTTP Responses Mule App Example
 
-NOTE: While a reason phrase is not defined for error responses, the connector attempts to define one based on the status code. Thus, a `Gateway Timeout` or `Internal Server Error` is returned in the scenarios explained before.
+The following example shows how to configure HTTP responses for the HTTP *Listener* source in Studio. The example is a Mule app flow where an endpoint stores logos for an account:
 
-[source,xml,linenums]
+* When the storage is successful, an HTTP regular response returns with status code `201`, reason phrase `Created`, and body as `Corporate Logo has been stored as a MuleSoft logo`.
+* When the storage fails, an HTTP error response returns where the status code is defined through the `errorCode` variable (if available) or a `500` code by default.
+** The custom header returns `X-Time`.
+** The body returns `Corporate Logo could not be stored`.
+** If there is a `CONNECTIVITY` error storing the logo, the returned status code is `504`, while any other errors result in a `500` status code.
```

Comment:
Can these lines be made parallel with the components of the description of success?

Reply:
Maybe we need a breakdown of 
* Success
** Status
** Reason
** Body
* Failure
** Status
** Reason
** Body